dev-python/imageio/imageio-1.5.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Video editing with Python"
```

Comment:
Shouldn't DESCRIPTION be "a Python library for reading and writing image data" or something similar? I assume you copy/pasted the ebuild from moviepy and forgot to update it.